pkg/apis/kubeone/types.go

```diff
@@ -63,6 +63,15 @@ type KubeOneCluster struct {
 	AssetConfiguration AssetConfiguration `json:"assetConfiguration,omitempty"`
 	// RegistryConfiguration configures how Docker images are pulled from an image registry
 	RegistryConfiguration *RegistryConfiguration `json:"registryConfiguration,omitempty"`
+	// LoggingConfig configures the Kubelet's log configuration
```

Comment:
I think this is a little bit better:

```suggestion
	// LoggingConfig configures the Kubelet's log rotation
```